cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -29,13 +28,6 @@
 ]
 
 
-@pytest.fixture(scope="function")
-def domain(request, appliance):
```

Comment:
https://github.com/ManageIQ/integration_tests/pull/8517/files#diff-11b93f3b87782fa7e6964884e93593f2R444

This fixture is still used by a test below - do you expect it to be covered by another global fixture somewhere?

Reply:
Nevermind, found the previous discussions on this that were resolved. I see its provided by another module.  DomainCollection is getting used otherwise directly as well.